app/views/bookmarks/_tools.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render Blacklight::Document::ActionsComponent.new(document: nil,
+                                                      tag: 'div',
+                                                      classes: "#{controller_name}Tools",
+                                                      link_classes: 'btn btn-outline',
```

Comment:
This partial override is specifically to change the classes passed into the initialization of the actions component to match prior Avalon behavior/rendering.